### ISSUE TITLE
Cannot specify project for google_sql_ssl_cert

### ISSUE THREAD

**Issue by toby-jn** [terraform-providers/terraform-provider-google#2988]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment
* If an issue is assigned to the "modular-magician" user, it is either in the process of being autogenerated, or is planned to be autogenerated soon. If an issue is assigned to a user, that user is claiming responsibility for the issue. If an issue is assigned to "hashibot", a community member has claimed the issue already.

### Terraform Version

```
Terraform v0.11.7
+ provider.google v1.20.0
+ provider.google-beta v1.20.0
+ provider.random v2.0.0
```
### Affected Resource(s)

* google_sql_ssl_cert

### Terraform Configuration Files

```tf
resource "google_sql_ssl_cert" "sql_client_cert" {
  provider    = "google-beta"
  common_name = "k8s-sql"
  instance    = "${google_sql_database_instance.sql_instance.name}"
}
```

### Expected Behavior

Resource should be created.

### Actual Behavior

```
Error: Error applying plan:

1 error(s) occurred:

* google_sql_ssl_cert.sql_client_cert: 1 error(s) occurred:

* google_sql_ssl_cert.sql_client_cert: project: required field is not set
```

if I try setting `project` in the resource:
```
Error: google_sql_ssl_cert.sql_client_cert: : invalid or unknown key: project
```

### Steps to Reproduce

1. create a `google_sql_ssl_cert` resource, without `project` set in the provider
1. `terraform apply`
1. apply fails due to missing `project` field

### Important Factoids

We create the project via terraform as a resource, so can't set it in the provider. If I manually set `project` in the provider everything works as expected.

**Comment by goobysnack** [comment]
Apologies, but I'm new to Terraform.  I'm getting both errors as well (the , and when I add the line:
```
resource "google_sql_ssl_cert" "clientssl" {
  project     = "${var.project_id}"
```
I get `google_sql_ssl_cert.clientssl: : invalid or unknown key: project`

I'm on google provider version 2.2.0 per
`Downloading plugin for provider "google" (2.2.0)...`

and Terraform version `v0.11.11`

Any ideas?

**Comment by chrisst** [comment]
This code was merged in yesterday which means this fix will be available in the next release, which should be 2.3.0.
